src/snapshot/tosvg.js

```diff
@@ -16,6 +16,9 @@ var Drawing = require('../components/drawing');
 var Color = require('../components/color');
 
 var xmlnsNamespaces = require('../constants/xmlns_namespaces');
+var DOUBLEQUOTE_REGEX = /"/g;
+var DUMMY_SUB = 'TOBESTRIPPED';
```

Comment:
OK. What's the solution then? 

- We could make the regex more precise [here](https://github.com/plotly/plotly.js/blob/master/src/snapshot/tosvg.js#L114) so that it grabs only `TOBESTRIPPED` inside `style=` props.
- Use another (more esoteric) dummy string
- Use another seriliazer than what's the browsers give us on the `window` object?